modules.d/99base/dracut-lib.sh

```diff
@@ -36,14 +36,14 @@ strstr() {
 # An empty $1 will not be considered matched, even if $2 is * which technically
 # matches; as it would match anything, it's not an interesting case.
 strglob() {
-    [ -n "$1" -a -z "${1##$2}" ]
+    [ -n "$1" -a -z "${1##"$2"}" ]
```

Comment:
This does not look good... read the function definition, the next code will fail if `$2` is surrounded by double quotes:

https://github.com/dracutdevs/dracut/blob/fd9cd02cb21f25d32fec4cd9889457ea6361f45e/modules.d/40network/net-lib.sh#L495